packages/web/src/utils/zIndex.ts

```diff
@@ -33,9 +33,10 @@ export enum zIndex {
   MUSIC_CONFETTI = 10000,
   WEB3_WALLET_CONNECT_MODAL = 10002,
   MODAL_OVERFLOW_MENU_POPUP = 10002,
-  USDC_MANUAL_TRANSFER_MODAL = 10002,
   ADD_FUNDS_MODAL = 10002,
   PREMIUM_CONTENT_PURCHASE_MODAL = 10002,
+  ADD_FUNDS_VENDOR_SELECTION_DRAWER = 10003,
```

Comment:
also fyi, was noticing the filter button popup menu wasn't displaying on web, might be a zindex issue.

Reply:
i fixed :)

Reply:
actually i did not fix. will look though